outscale/data_source_outscale_account.go

```diff
@@ -0,0 +1,155 @@
+package outscale
+
+import (
+	"context"
+	"time"
+
+	oscgo "github.com/outscale/osc-sdk-go/v2"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"github.com/terraform-providers/terraform-provider-outscale/utils"
+)
+
+func dataSourceAccount() *schema.Resource {
+	return &schema.Resource{
+		Read: dataSourceAccountRead,
+		Schema: map[string]*schema.Schema{
+			"account_id": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"additional_emails": {
+				Type:     schema.TypeList,
+				Computed: true,
+				Elem:     &schema.Schema{Type: schema.TypeString},
+			},
+			"city": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"company_name": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"country": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"customer_id": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"email": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"first_name": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"job_title": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"last_name": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"mobile_number": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"phone_number": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"state_province": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"vat_number": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"zip_code": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"request_id": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+		},
+	}
+}
+
+func dataSourceAccountRead(d *schema.ResourceData, meta interface{}) error {
+
+	conn := meta.(*OutscaleClient).OSCAPI
+
+	req := oscgo.ReadAccountsRequest{}
+
+	var resp oscgo.ReadAccountsResponse
+	var err error
+
+	err = resource.Retry(30*time.Second, func() *resource.RetryError {
+		resp, _, err = conn.AccountApi.ReadAccounts(context.Background()).ReadAccountsRequest(req).Execute()
+		if err != nil {
+			return utils.CheckThrottling(err)
+		}
+		return nil
+	})
+	if err != nil {
+		return err
+	}
+
+	account := resp.GetAccounts()[0]
+
```

Comment:
Can you check please if you got more than one element in the read response please?